Snakefile

```diff
@@ -129,6 +130,15 @@ rule collect_summaries:
 
 ################# Plots #################
 
+rule plot_package_degree:
+    input:
+        "package-data/all.tsv"
+    output: "plots/package_degrees.svg"
```

Comment:
Can you adhere to the style of the rest of the file here (new line and indent)?

Reply:
fixed